clients/java/client/src/main/java/org/camunda/bpm/client/interceptor/impl/RequestInterceptorHandler.java

```diff
@@ -24,7 +24,6 @@
 import org.camunda.bpm.client.impl.EngineClientLogger;
 import org.camunda.bpm.client.impl.ExternalTaskClientLogger;
 import org.camunda.bpm.client.interceptor.ClientRequestInterceptor;
-
```

Comment:
I think you can revert all changes of this class, so the commit for the PR is clean in the end.

Reply:
Yes, you are right. reverted 
thanks